Sources/XcodeProj/Extensions/String+md5.swift

```diff
@@ -21,28 +21,29 @@
  */
 
 import Foundation
+import CommonCrypto
 
 extension String {
-    var md5: String {
-        if let data = data(using: .utf8, allowLossyConversion: true) {
-            let message = data.withUnsafeBytes { bufferPointer in
-                Array(UnsafeBufferPointer(
-                    start: bufferPointer.baseAddress?.assumingMemoryBound(to: UInt8.self),
-                    count: data.count
-                ))
-            }
-
-            let MD5Calculator = MD5(message)
-            let MD5Data = MD5Calculator.calculate()
+    private func toHex(_ char: UInt8) -> UInt8 {
+        if char < 10 {
```

Comment:
Would you mind adding a comment explaining the rationale behind comparing char with 10?